tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noEmitOnError": true,
-    "lib": ["es2017", "dom"],
+    "lib": ["es2017", "dom", "dom.iterable"],
```

Comment:
cool! should we also add this to the main ts template?

Reply:
Probably, yeah. I'll create an MR to add it there too.